Check style is failing

E.g. this build https://travis-ci.org/jaegertracing/jaeger-client-java/jobs/285212390

```
:jaeger-core:checkstyleMain[ant:checkstyle] [ERROR] /home/travis/build/jaegertracing/jaeger-client-java/jaeger-core/src/main/java/com/uber/jaeger/SpanContext.java:179: Abbreviation in name 'getTraceID' must contain no more than '2' consecutive capital letters. [AbbreviationAsWordInName]
[ant:checkstyle] [ERROR] /home/travis/build/jaegertracing/jaeger-client-java/jaeger-core/src/main/java/com/uber/jaeger/SpanContext.java:187: Abbreviation in name 'getSpanID' must contain no more than '2' consecutive capital letters. [AbbreviationAsWordInName]
[ant:checkstyle] [ERROR] /home/travis/build/jaegertracing/jaeger-client-java/jaeger-core/src/main/java/com/uber/jaeger/SpanContext.java:195: Abbreviation in name 'getParentID' must contain no more than '2' consecutive capital letters. [AbbreviationAsWordInName]
```

We keep the methods like `getSpanID` as deprecated for legacy reasons. We also don't specify the checkstyle config directly, iirc it's imported from one of the google jars.  Can we still suppress the checks on these 3 methods?

cc @pavolloffay @objectiser @vprithvi

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -17,13 +17,10 @@ We gratefully welcome improvements to documentation as well as to code.
 ### Code Style
 
 This project uses [google java style](https://google.github.io/styleguide/javaguide.html) configured 
-via [checkstyle/google_checks.xml](https://github.com/checkstyle/checkstyle/blob/master/src/main/resources/google_checks.xml)
-
-You can also setup the [IntelliJ plugin](https://plugins.jetbrains.com/plugin/8527)
-to reformat code from within your IDE or import code style settings from 
-[google/intellij-java-google-style](https://github.com/google/styleguide/blob/gh-pages/intellij-java-google-style.xml)
+via [config/checkstyle](./config/checkstyle).
 
 ### Lombok
+
 This project uses [Lombok](https://projectlombok.org/) to reduce boilerplate. You can setup
 the [IntelliJ plugin](https://plugins.jetbrains.com/plugin/6317) to add IDE support. 
 

--- a/jaeger-thrift/README.md
+++ b/jaeger-thrift/README.md
@@ -2,3 +2,5 @@
 
 This module builds a jar containing auto-generated Thrift classes
 for the IDL defined in https://github.com/uber/jaeger-idl.
+
+It uses Thrift compiler from a Docker image, thus requires a working Docker installation.

--- a/jaeger-thrift/build.gradle
+++ b/jaeger-thrift/build.gradle
@@ -11,12 +11,6 @@ dependencies {
     signature 'org.codehaus.mojo.signature:java16:1.1@signature'
 }
 
-/**
- * Note that you need to install thrift 0.9.2 on your system for this task to work.
- *
- * If you are using macOS, you can install this specific version via homebrew like so:
- * `brew install https://raw.githubusercontent.com/Homebrew/homebrew-core/56d8c1eba1e5ac30290dd0c486f4bba37f821e42/Formula/thrift.rb`
- */
 compileThrift {
     thriftExecutable "${projectDir}/../travis/docker-thrift/thrift"
     sourceDir "${projectDir}/../idl/thrift"

